packages/jellyfish-transaction/__tests__/script/index.test.ts

```diff
@@ -725,3 +725,165 @@ describe('P2WSH: [OP_0, OP_PUSHDATA<SHA256(script)>]', () => {
     expect(smartBuffer.toBuffer().toString('hex')).toStrictEqual(hex)
   })
 })
+
```

Comment:
Move all these to the line after '0160' and before '016a'.

Reply:
Also, check other sections too. It should be arranged in this order => 0160, 0161 ... 0169, 016a